zipline/_protocol.pyx

```diff
@@ -76,6 +81,289 @@ cdef class BarData:
             self.data_frequency
         )
 
+    def spot_value(self, assets, fields):
+        """
+        Returns the spot value of the given assets for the given fields
+        at the current simulation time.  Spot values are the as-traded price
```

Comment:
yeah.  but the vast majority of the time, spot values aren't adjusted, and I want to reinforce that to users.  I'll fiddle with the wording.